tensorly/decomposition/tests/test_cmtf_als.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+
+import tensorly as tl
+from ..cmtf_als import (factor_match_score_3d, cmtf_als_for_third_order_tensor)
+from ...testing import assert_array_almost_equal, assert_array_less
+
+
+def test_factor_match_score_3d():
+    # same factors
+    X_true = tl.kruskal_tensor.KruskalTensor((None, [tl.tensor([[4, 1], [0, 2]]), tl.tensor([[0, 1], [3, 5]]),
```

Comment:
I think in the function that gets the parameters, it is shorter if we only give for Kruskal tensors instead of four tuples, isn't it? Especially if I import KruskalTensor in the beginning